hr-time/resources/unload.js

```diff
@@ -0,0 +1,51 @@
+const syncDelay = ms => {
+  const start = performance.now();
+  let elapsedTime;
+  do {
+    elapsedTime = performance.now() - start;
+  } while (elapsedTime < ms);
```

Comment:
const start = performance.now();
while (performance.now() < start + ms);
const elapsedTime = performance.now();

Is a bit cleaner.